app/controllers/DashboardController.scala

```diff
@@ -74,19 +76,20 @@ class DashboardController @Inject() (
                 renderDashboard(page, dashboardData, pensionSchemeDetails, lockWarning)
               case Some(transfers) =>
                 transfers.map {
+                  val owner =
+                    request.authenticatedUser match {
+                      case PsaUser(psaId, _, _, _) => psaId.value
+                      case PspUser(pspId, _, _, _) => pspId.value
+                    }
+
                   transfer =>
-                    (transfer.transferReference, transfer.qtReference) match {
-                      case (Some(transferRef), None)              =>
+                    transfer.transferId match {
```

Comment:
So the idea is that we move away from passing a qtRef or transferId around to one unified id which can either be a qtRef or a transferId?